docs/cip/README.md

```diff
@@ -4,518 +4,1145 @@ Title: Ouroboros Leios - Greater transaction throughput
+version extends Praos with optional secondary blocks and committee validation
+to achieve significant increases in throughput while
+maintaining familiar transaction semantics.
+
+The Cardano Problem Statement [CPS-18 Greater Transaction Throughput][cps-18]
+further motivates the need for higher transaction throughput and marshals
+quantitative evidence of existing mainnet bottlenecks. Realizing higher
+transaction rates is also necessary for long-term Cardano techno-economic
+viability as rewards contributions from the Reserve pot diminish: fees from more
+transactions will be needed to make up that deficit and keep sound the finances
+of stakepool operations.
+
+A major protocol upgrade like Leios will take
+significant time to implement, test, and audit, it is important to began
+implementation well before transaction demand on mainnet exceeds the
+capabilities of Ouroboros Praos. The plot below shows the historically
+diminishing rewards and a forecast of their continued reduction: the forecast is
+mildly uncertain because the future pattern of staking behavior, transaction
+fees, and node efficiency might vary considerably.
+
+<div align="center">
+<a name="figure-1" id="figure-1"></a>
+<p>
+  <img src="images/reward-forecast-bau.svg" alt="Forecast of rewards on Cardano mainnet">
+</p>
+
+_Figure 1: Forecast of rewards on Cardano mainnet_
+
+</div>
+
+Ouroboros Praos cannot support the high transaction volume needed to generate
+the fees that will eventually be needed to offset the diminishing rewards.
+However, as sustained throughput of transactions grows beyond 50 transactions
+per second, there is more opportunity for simultaneously reducing fees,
+augmenting the Treasury, and increasing SPO and delegator rewards.
+
+<div align="center">
+<a name="figure-2" id="figure-2"></a>
+<p>
+  <img src="images/spo-profitability.svg" alt="SPO profitability under Praos, as a function of transaction volume">
+</p>
+
+_Figure 2: SPO profitability under Praos, as a function of transaction volume_
+
+</div>
+
+The Leios protocol specified in this document represents a balance
+between immediate scalability needs and long-term protocol evolution. The
+approach prioritizes practical deployment and ecosystem compatibility while
+establishing the foundation for subsequent protocol versions with higher
+throughput capacity.
+
+## Specification
+
+Leios extends Ouroboros Praos by adding a second operating mode. The protocol switches between these two modes automatically based on network conditions and whether certificates appear in blocks. This lets the protocol keep all of Praos's security properties while processing more transactions when the network can handle it.
+
+<div align="center">
+  <a name="figure-3" id="figure-3"></a>
+  <p name="dual-mode-figure">
+    <img src="images/dual-mode-simplified.svg" alt="Dual-Mode Operation">
+  </p>
+
+  <em>Figure 3: Ouroboros Leios Dual-Mode Operation</em>
```

Comment:
Sometimes we're using `_` for figure captions, but here we use `<em>`. I'm not sure if consistency matters, but in might make later semi-automated reformatting easier.